rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2899,17 +2897,14 @@ namespace rsx
 		else if (capture_current_frame)
 		{
 			capture_current_frame = false;
-			std::stringstream os;
-			cereal::BinaryOutputArchive archive(os);
-			const std::string& filePath = fs::get_config_dir() + "captures/" + Emu.GetTitleID() + "_" + date_time::current_time_narrow() + "_capture.rrc";
-			archive(frame_capture);
-			{
-				// todo: may want to compress this data?
-				fs::file f(filePath, fs::rewrite);
-				f.write(os.str());
-			}
 
-			rsx_log.success("capture successful: %s", filePath.c_str());
+			const std::string file_path = fs::get_config_dir() + "captures/" + Emu.GetTitleID() + "_" + date_time::current_time_narrow() + "_capture.rrc";
+			const std::string file_data = cereal_serialize(frame_capture);
+
+			// todo: may want to compress this data?
+			fs::file(file_path, fs::rewrite).write(file_data);
```

Comment:
If you already changing it, use fs::write_file to avoid crash on failure to create file. Maybe log an error about it.